naturebridge.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+---
+### Venue information
+
+2019 Ruby by the Bay will be held March 28 - April 1 (Friday - Monday) at NatureBridge in Sausalito, California.
+
+![Aerial photo of venue next to the ocean](/images/naturebridge_from_above.jpg "Aerial photo of venue next to the ocean"){:class="img-responsive"}
+
+###### What is this beautiful place??
+
+![Photo of NatureBridge signage in front of some of their buildings](/images/naturebridge_sign.jpg "Photo of NatureBridge signage in front of some of their buildings"){:class="img-responsive"}
+
+NatureBridge is a nonprofit that focuses on educating kids about the environment.  They have school groups during the school year, and run day camps during the summers.
+
+###### Seriously how is it this beautiful??
+
+NatureBridge is lucky enough to lease their space from the National Park Service. It's housed in Golden Gate National Recreation Area. The campus was originally created by the Army.
+
+###### What will sleeping arrangements be like?
+We have four bunkhouses:
+* Non-binary people (Owl's Roost)
+* Women's (Heron)
+* Men's (Pelican)
+* All genders (Raven)
+
+All bunkhouses have shockingly clean bathrooms attached.  Toilets are in stalls and showers have curtains.  Some of the spaces may have (coin-operated) laundry available.
+
+The bunkhouses have hot water and are heated.
+
+![Photo showing off view of the lagoon from the bunks](/images/naturebridge_bunks_and_view.jpg "Photo showing off view of the lagoon from the bunks"){:class="img-responsive"}
+
+Many of the bunks have fantastic views of the outdoors!  Some bunks have power outlets available; please let us know if you require a power outlet for your CPAP and we'll help make sure you get an appropriate bed.
```

Comment:
Maybe we could change this to say "please let us know if you require a power outlet and we will do our best to accommodate"

I don't have a CPAP but if I was staying there I'd need an outlet so I could pump.  Other people might have various reasons.  I think it's better if we leave it a little more open ended so that people can explain their needs, there may be reasons we can't think of too....